tests/test_goodconf.py

```diff
@@ -208,11 +207,13 @@ def test_precedence(tmpdir):
     path = tmpdir.join("myapp.json")
     path.write(json.dumps({"init": "file", "env": "file", "file": "file"}))
 
-    class TestConf(GoodConf, default_files=[path]):
```

Comment:
Does this no longer work? The default Pydantic docs still show configuration via class arguments as supported: https://docs.pydantic.dev/2.8/api/config/

Reply:
https://github.com/pydantic/pydantic-settings/blob/4840d69da15ffca3063b866679fdabedc62f350c/pydantic_settings/main.py#L71-L77

shows what is needed to keep this working.

Reply:
Thanks for looking, and for the link @apollo13!
This should be updated to support configuration via class arguments